src/common/feedGenerator.ts

```diff
@@ -56,31 +57,53 @@ export const whereKeyword = (
   return `EXISTS${query}`;
 };
 
+export const getExcludedAdvancedSettings = async (
```

Comment:
I think your previous implementation was better.
Now you introduce more database queries with no potential performance gain because we still have to wait for the features request.
Why did you introduce this change?

Reply:
I realized an edge case with the previous implementation when the scenario has the `defaultEnabledState` as false and overriding it to true. In the previous first condition, it would meet the requirement which when applying the additional where conditions it may not meet the additional requirements but since we use `OR` it still became valid.

The condition has to be strict (based on priorities) and check each advanced settings available which led me to merge through the backend.

Reply:
Ok, let's let it be for now. Once we finish the experiment we may think of other ways :)